views/timeslot.tmpl

```diff
@@ -19,16 +19,35 @@
 {{define "content"}}
 <div class="container-fluid current-next">
   <div class="container">
-  {{if .PageData.MixcloudAvailable}}
+  {{if ne .PageData.ODState 0}}
     <div class="row justify-content-center">
-      <div class="col-11 col-md-10 py-2 mixcloud">
+      <div class="col-11 col-md-10 py-2 mixcloud text-center text-lg-left">
         <h2>
           <a href="/ontap/" title="Goto {{.PageContext.ShortName}} {{.PageContext.ODName}}">
             {{- .PageContext.ShortName}} {{.PageContext.ODName -}}
-          </a>: Listen Again
+          </a>
+          {{- if ge .PageData.ODState 3 -}}: Listen Again{{end}}
         </h2>
-        <iframe width="100%" height="120" src="//www.mixcloud.com/widget/iframe/?feed={{.PageData.Timeslot.MixcloudStatus}}&hide_cover=1&light=1" frameborder="0"></iframe>
-        <a href="/ontap/" title="Goto {{.PageContext.ShortName}} {{.PageContext.ODName}}">Listen to more shows on demand...</a>
+        {{if eq .PageData.ODState 1}}
+          <h3>This show starts on {{.PageData.Timeslot.StartTime.Format "Monday, _2 Jan 2006 at 15:04"}}.</h3>
+          <p class="h5">Just can't wait? Check out previous shows <a href="{{.PageData.Timeslot.Season.ShowMeta.MicroSiteLink.URL}}">{{.PageContext.ODName -}}...</a></p>
```

Comment:
why is this css class a thing? why not use ...h5?

Reply:
I used this because I think from a web standpoint that it is technically a paragraph element, but I wanted it to be bigger for the style...